src/components/layouts/docs-inner.njk

```diff
@@ -27,5 +27,5 @@
         <p>Read the full documentation on our side navigation on the component page.</p>
       </div>
```

Comment:
Is the order correct here? `<main>` doesn't seem to have any divs inside.

Reply:
Yeah `<main>` isn't a parent to any divs in this refactor